build.sbt

```diff
@@ -1,27 +1,21 @@
 import sbtrelease._
 import ReleaseStateTransformations._
 
-releaseSettings
-
-sonatypeSettings
-
 organization := "com.gu"
 
 name := "crosswords-api-client"
 
-scalaVersion := "2.11.4"
-
-crossScalaVersions := Seq("2.10.4", " 2.11.4")
+scalaVersion := "2.11.7"
 
 resolvers ++= Seq(
   "Guardian GitHub Releases" at "http://guardian.github.io/maven/repo-releases",
   "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"
 )
 
 libraryDependencies ++= Seq(
-  "com.typesafe.play" %% "play-json" % "2.3.7",
-  "joda-time" % "joda-time" % "2.3",
-  "org.specs2" %% "specs2" % "2.3.11" % "test"
+  "com.typesafe.play" %% "play-json" % "2.5.3",
+  "joda-time" % "joda-time" % "2.9.3",
+  "org.specs2" %% "specs2-core" % "3.6" % "test"
```

Comment:
[Looks like there are lots more versions out for this library now](http://notes.implicit.ly/tagged/specs2). That said, maven only shows some of them. Maybe 3.7 is a better choice with that in mind and if that doesn't work there's a 3.3.1 point release.